dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/context/WorkContext.java

```diff
@@ -106,6 +106,14 @@
      */
     private final Map<String, ProgramStageInstance> programStageInstanceMap;
 
+    /**
+     * Holds a Map of all {@see ProgramStageInstance} associated to the Events
+     * to import and only constains already persisted values.
+     *
+     * Map: key -> ProgramStageInstance UID value -> ProgramStageInstance
+     */
+    private final Map<String, ProgramStageInstance> persistedProgramStageInstanceMap;
```

Comment:
I would remove any type names from variable names. That makes them more verbose than they already are in Java. The IDE lets us know this is a map anyway. So maybe `persistedProgramStageInstances`